diff -Naru dgamelaunch.orig/dgl-create-chroot dgamelaunch/dgl-create-chroot
--- dgamelaunch.orig/dgl-create-chroot	2022-06-01 01:24:05.636453579 +0000
+++ dgamelaunch/dgl-create-chroot	2022-06-01 04:39:27.815179632 +0000
@@ -10,7 +10,7 @@
 
 
 # Same as chroot_path in dgl config file
-CHROOT="/opt/nethack/nethack.alt.org/"
+CHROOT="/home/nethack/"
 # the user & group from dgamelaunch config file.
 USRGRP="games:games"
 # COMPRESS from include/config.h; the compression binary to copy. leave blank to skip.
@@ -20,9 +20,9 @@
 # fixed data to copy (leave blank to skip)
 #NH_PLAYGROUND_FIXED="/home/paxed/hacking/coding/nethacksource/nethack-3.4.3-nao/nh343/"
 # HACKDIR from include/config.h; aka nethack subdir inside chroot
-NHSUBDIR="/nh343/"
+NHSUBDIR="/nh366/"
 # VAR_PLAYGROUND from include/unixconf.h
-NH_VAR_PLAYGROUND="/nh343/var/"
+NH_VAR_PLAYGROUND="/nh366/var/"
 
 # only define this if dgl was configured with --enable-sqlite
 SQLITE_DBFILE="/dgldir/dgamelaunch.db"
@@ -91,9 +91,9 @@
 cp "$CURDIR/dgamelaunch" "$DGLFILE"
 ln -s "$DGLFILE" dgamelaunch
 
-mkdir -p "$CHROOT/dgldir/inprogress-nh343"
+mkdir -p "$CHROOT/dgldir/inprogress-nh366"
 mkdir -p "$CHROOT/dgldir/userdata"
-chown "$USRGRP" "$CHROOT/dgldir/inprogress-nh343"
+chown "$USRGRP" "$CHROOT/dgldir/inprogress-nh366"
 chown "$USRGRP" "$CHROOT/dgldir/userdata"
 
 
@@ -145,8 +145,8 @@
 cp "$CURDIR/examples/dgl_menu_main_user.txt" .
 cp "$CURDIR/examples/dgl_menu_watchmenu_help.txt" .
 cp "$CURDIR/examples/dgl-banner" .
-cp "$CURDIR/dgl-default-rcfile" "dgl-default-rcfile.nh343"
-chmod go+r dgl_menu_main_anon.txt dgl_menu_main_user.txt dgl-banner dgl-default-rcfile.nh343
+cp "$CURDIR/dgl-default-rcfile" "dgl-default-rcfile.nh366"
+chmod go+r dgl_menu_main_anon.txt dgl_menu_main_user.txt dgl-banner dgl-default-rcfile.nh366
 
 NHSUBDIR="`echo ${NHSUBDIR%/}`"
 NHSUBDIR="`echo ${NHSUBDIR#/}`"
diff -Naru dgamelaunch.orig/examples/dgamelaunch.conf dgamelaunch/examples/dgamelaunch.conf
--- dgamelaunch.orig/examples/dgamelaunch.conf	2022-06-01 01:24:05.636453579 +0000
+++ dgamelaunch/examples/dgamelaunch.conf	2022-06-01 04:35:42.132782799 +0000
@@ -17,7 +17,7 @@
 # Max length for newly registered nicks. Must be less than 20.
 # By default, NetHack only stores the first 10 chars of a name into
 # the record/logfile.
-maxnicklen = 10
+maxnicklen = 20
 
 # Set the default watching-screen sorting mode. Can be one of
 # "username", "game", "windowsize", "starttime" or "idletime".
@@ -41,7 +41,7 @@
 #                 ]
 
 # Path to a prepared chroot jail.
-chroot_path = "/opt/nethack/nethack.alt.org/"
+chroot_path = "/home/nethack/"
 
 # From inside the jail, dgamelaunch's working directory for rcfiles/ttyrec/etc
 dglroot = "/dgldir/"
@@ -102,7 +102,7 @@
 # before dgl exits. Default value is 0, which disables the idling timer.
 # Does not apply to external programs or config editors.
 # For setting game idle time, use max_idle_time in the game DEFINE.
-# menu_max_idle_time = 1024
+menu_max_idle_time = 180
 
 # Passwd refers to the file that stores the user database.
 # The default passwd file is "/dgl-login" for flat-text database, and for
@@ -234,9 +234,9 @@
         commands["c"] = chpasswd
         commands["e"] = chmail
         commands["w"] = watch_menu
-        commands["o"] = ifnxcp "/dgl-default-rcfile.nh343" "%ruserdata/%n/%n.nh343rc",
-		        exec "/bin/virus" "%ruserdata/%n/%n.nh343rc"
-        commands["p"] = play_game "NH343"
+        commands["o"] = ifnxcp "/dgl-default-rcfile.nh366" "%ruserdata/%n/%n.nh366rc",
+		        exec "/bin/virus" "%ruserdata/%n/%n.nh366rc"
+        commands["p"] = play_game "NH366"
         commands["q"] = quit
 }
 
@@ -317,18 +317,18 @@
 #
 
 DEFINE {
-  game_path = "/nh343/nethack"
-  game_name = "NetHack 3.4.3"
-  short_name = "NH343"
+  game_path = "/nh366/jnethack"
+  game_name = "JNetHack 3.6.6"
+  short_name = "NH366"
 
-  game_args = "/nh343/nethack", "-u", "%n"
+  game_args = "/nh366/jnethack", "-u", "%n"
 
   spooldir = "/mail/"
-  rc_template = "/dgl-default-rcfile.nh343"
+  rc_template = "/dgl-default-rcfile.nh366"
 
-  rc_fmt = "%ruserdata/%n/%n.nh343rc"
+  rc_fmt = "%ruserdata/%n/%n.nh366rc"
 
-  inprogressdir = "%rinprogress-nh343/"
+  inprogressdir = "%rinprogress-nh366/"
 
 # The place where ttyrecs are stored for this game.
 # If this is not defined, ttyrecs are not saved for this game.
@@ -337,9 +337,9 @@
 
 
   # back up savefile
-  commands = cp "/nh343/var/save/%u%n.gz" "/nh343/var/save/%u%n.gz.bak",
+  commands = cp "/nh366/var/save/%u%n.gz" "/nh366/var/save/%u%n.gz.bak",
   # set NETHACKOPTIONS to point to the rcfile
-             setenv "NETHACKOPTIONS" "@%ruserdata/%n/%n.nh343rc",
+             setenv "NETHACKOPTIONS" "@%ruserdata/%n/%n.nh366rc",
   # set up nethack mail stuff, assuming it's compiled with it...
              setenv "MAIL" "/mail/%n",
              setenv "SIMPLEMAIL" "1",
diff -Naru dgamelaunch.orig/examples/dgl_menu_main_user.txt dgamelaunch/examples/dgl_menu_main_user.txt
--- dgamelaunch.orig/examples/dgl_menu_main_user.txt	2022-06-01 01:24:05.636453579 +0000
+++ dgamelaunch/examples/dgl_menu_main_user.txt	2022-06-01 04:35:42.132782799 +0000
@@ -12,7 +12,7 @@
  e) Change email address
  w) Watch games in progress
  o) Edit options
- p) Play NetHack 3.4.3
+ p) Play JNetHack 3.6.6
  q) Quit
 
  =>
